docs/src/main/paradox/docs/delta/index.md

```diff
@@ -7,3 +7,33 @@
 @@@
 
 # Nexus Delta
+
+Blue Brain Nexus Delta is a low latency, scalable and secure system that realizes a range of functions to support
+data management and knowledge graph lifecycles.
+
+It is a central piece of the Nexus ecosystem of software components as it offers a set of foundational capabilities to
```

Comment:
Bullet points instead of "It is" repetition ?

Reply:
normally I would agree, but not for this page. I've added the name instead of one of the it is.